tests/functional/a/arguments_renamed.py

```diff
@@ -40,15 +40,15 @@ class Child(Parent):
     def test(self, arg1): # [arguments-renamed]
         return arg1 + 1
 
-    def kwargs_test(self, arg, *, value1, var2): #[arguments-renamed]
+    def kwargs_test(self, arg, *, value1, var2): #[arguments-differ]
         print(f"keyword parameters are {value1} and {var2}.")
 
 class Child2(Parent):
 
     def test(self, var): # [arguments-renamed]
         return var + 1
 
-    def kwargs_test(self, *, var1, kw2): #[arguments-renamed, arguments-differ]
+    def kwargs_test(self, *, var1, kw2): #[arguments-differ]
```

Comment:
What do we want to do here? We go from:
`def kwargs_test(self, arg, *, var1, var2):`
to
`def kwargs_test(self, *, var1, kw2):`

Is the `arguments-renamed` or `arguments-differ`?

Reply:
I think arguments-differ is an improvement in this case. For kw-only args, an argument cannot truly be "renamed".